modules/core/src/file-utils/load-file.js

```diff
@@ -8,7 +8,7 @@ export function loadFile(url, loaders, options) {
 
 export function loadFileSync(url, loaders, options) {
   const data = readFileSync(url, options);
-  const result = parseFileSync(data, url, loaders, options);
+  const result = parseFileSync(data, loaders, options, url);
```

Comment:
Add a comment on when we should use `url`?

Reply:
Better yet, I updated the docs